src/view/com/util/MainScrollProvider.tsx

```diff
@@ -58,13 +46,6 @@ export function useOnMainScroll(): [OnScrollHandler, boolean, ResetCb] {
   const onScroll = useCallback(
     (e: NativeScrollEvent) => {
       'worklet'
-      // Keep track of whether we want to show "scroll to top".
-      if (!isScrolledDown && e.contentOffset.y > s.window.height) {
-        runOnJS(setIsScrolledDown)(true)
-      } else if (isScrolledDown && e.contentOffset.y < s.window.height) {
```

Comment:
This logic is in `List` now and it's comparing to the threshold that was hardcoded in the pager (`200`). We could make that customizable or choose a number in the middle. Regardless, `s.window.height` didn't quite make sense because it's frozen at startup and doesn't take window resize into account.